src/test/java/harness/TestCommand.java

```diff
@@ -26,7 +27,8 @@
  */
 public class TestCommand {
 
-  private static ObjectMapper objectMapper = new ObjectMapper();
+  private static ObjectMapper objectMapper =
```

Comment:
while you are here, can you change it to private final static ObjectMapper OBJECT_MAPPER

Reply:
Done